chrome/options.browserify.js

```diff
@@ -0,0 +1,120 @@
+var settings = {
+  autoSubmit: {
+    type: "checkbox",
+    title: "Automatically submit forms after filling",
+    value: false
+  },
+  use_fuzzy_search: {
+    type: "checkbox",
+    title: "Use fuzzy search",
+    value: true
+  },
+  customStore: {
+    title: "Custom password store locations",
+    value: [{name: "one", enabled: true, name: "fish", path: "/home/fish"}],
```

Comment:
This default entry "fish" is confusing, the first time I saw it I toggled the checkbox twice, closed, and browserpass started showing me error `Error: lstat /home/fish: no such file or directory`. I will change this line to `value: [{}]` if you don't have objections - the placeholders explain well what needs to be entered where.

Reply:
Oops - that was a leftover test value. I've just nuked it. Thanks for picking it up :-).

Reply:
(To clarify - the default is supposed to be an empty array; it doesn't need to contain a value).

Reply:
I tried a few options before writing that comment, and actually I found it quite pleasant for the eye to have one empty line pre-created

Reply:
Makes sense - I've added your suggested default in f686716.